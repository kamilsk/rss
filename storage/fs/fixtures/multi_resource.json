{
  "id": "multi_resource",
  "name": "multi resource",
  "src": [
    {"src": "https://rss.octolab.net/kamilsk/podcasts"},
    {"src": "https://rss.octolab.net/kamilsk/releases"},
    {"src": "https://rss.octolab.net/octolab/releases"}
  ]
}
