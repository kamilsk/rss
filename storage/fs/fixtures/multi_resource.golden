{"id":"multi_resource","name":"multi resource","src":[{"name":"end resource","src":"https://rss.octolab.net/kamilsk/podcasts"},{"name":"end resource","src":"https://rss.octolab.net/kamilsk/releases"},{"name":"end resource","src":"https://rss.octolab.net/octolab/releases"}]}
